_episodes_rmd/08-plot-ggplot2.Rmd

```diff
@@ -40,53 +40,67 @@ it is the most effective for creating publication quality
 graphics.
 
 ggplot2 is built on the grammar of graphics, the idea that any plot can be
-expressed from the same set of components: a **data** set, a
-**coordinate system**, and a set of **geoms**--the visual representation of data
-points.
-
-The key to understanding ggplot2 is thinking about a figure in layers.
-This idea may be familiar to you if you have used image editing programs like Photoshop, Illustrator, or
-Inkscape.
-
-Let's start off with an example:
+built from the same set of components: a **data set**,
+**mapping aesthetics**, and graphical **layers**:
+
+* **Data sets** are the data that you, the user, provide.
+
+* **Mapping aesthetics** are what connect the data to the graphics.
+They tell ggplot2 how to use your data to affect how the graph looks,
+such as changing what is plotted on the X or Y axis, or the size or
+color of different data points.
+
+* **Layers** are the actual graphical output from ggplot2. Layers
+determine what kinds of plot are shown (scatterplot, histogram, etc.),
+the coordinate system used (rectangular, polar, others), and other
+important aspects of the plot. The idea of layers of graphics may
+be familiar to you if you have used image editing programs
+like Photoshop, Illustrator, or Inkscape.
+
```

Comment:
Second major change (below): Broke the first example apart so that it's built up in stages (data, aesthetics, geoms) rather than putting in a large block of code all at once. In my experience this helps learners understand better what each component is doing and how they fit together.